app/dashboard/static/js/app/view-socs-soc-job-kernel.2017.4.js

```diff
@@ -701,10 +536,15 @@ define([
             });
     });
 
-    gSessionStorage = storage('soc-' + gSoc + '-' + gJob + '-' + gKernel);
-    gResultFilter = filter('data-filter');
-    getBoots();
-    registerEvents();
+
+    gPlansTable = table({
+        tableId: 'planstable',
+        tableLoadingDivId: 'table-loading',
+        tableDivId: 'table-div',
+    });
+
+    getPlans();
```

Comment:
Maybe add timeouts like on other views?  Is there any good reason to not have timeouts here?